kubevirt-101/step1.md

```diff
@@ -17,14 +17,14 @@ Run the following command to deploy the KubeVirt Operator:
 
 `kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml`{{execute}}
 
-This demo environment already runs within a virtualized environment, and in order to be able to run VMs here we need to pre-configure KubeVirt so it uses software-emulated virtualization instead of trying to use real hardware virtualization.
-
-`kubectl create configmap kubevirt-config -n kubevirt --from-literal debug.useEmulation=true`{{execute}}
-
 Now let's deploy KubeVirt by creating a Custom Resource that will trigger the 'operator' reaction and perform the deployment:
 
 `kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml`{{execute}}
 
+This demo environment already runs within a virtualized environment, and in order to be able to run VMs here we need to configure KubeVirt so it uses software-emulated virtualization instead of trying to use real hardware virtualization.
```

Comment:
The sentence is still very wordy.  How about shortening it?

`This demo environment runs within a virtualized environment.  To run VMs here we need to configure KubeVirt to use software-emulated virtualization instead of normal hardware virtualization.`

Reply:
I could, I just moved it from one place to another...

Reply:
I made a change.